apps/gportal/src/components/Datasets.vue

```diff
@@ -77,60 +65,31 @@ function setRefs(value) {
   }
 }
 
-function selectAll () {
-  checkboxes.value.forEach(value => {
+function selectAll() {
+  checkboxes.value.forEach((value) => {
     if (selection.value.indexOf(value) === -1) {
       selection.value.push(value);
     }
   });
   setUrl();
 }
 
+onMounted(() => {
```

Comment:
This is really bad hack. Is there a slot or prop available to pass a column header?

Reply:
not yet but added one